src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -1686,7 +1970,22 @@ public void WriteConnectionString(ApplicationModel.IResource resource) { }
         public System.Threading.Tasks.Task WriteEnvironmentVariablesAsync(ApplicationModel.IResource resource) { throw null; }
     }
 
-    public sealed partial class PublishingOptions
+    public sealed partial class PublishingActivity
```

Comment:
Experimental

Reply:
@mitchdenny

Reply:
It is marked experimental and has been since last week:
https://github.com/dotnet/aspire/blame/1007a6c69ab3e2e1a09e06bbf02d53a1208a8c2e/src/Aspire.Hosting/Publishing/PublishingActivityProgressReporter.cs#L14